### DISCUSSION TITLE
BoJ added in Karazhan alone will not work to increase its popularity

### DISCUSSION THREAD

**Discussion by gabonvaricel** [chromiecraft/chromiecraft#7186] [Suggestions]
I saw someone write here a suggestion to make bosses in kara drop BoJ to increase the popularitt of the instance. But that alone will not work because the badge gear on the vendor right now  sucks ass. This will only work once ZA and Sunwell badge gear is added.

**Comment by 55Honey** [comment]
Thanks for your suggestion. You should consider posting as a reply in the related thread instead of opening a new one.
